Dashboard: Administracion de una sala.

manuelitod opened the pull request:
Como administrador ahora se puede editar la imagen, los datos y los componentes de una sala.

Como fueron probado los cambios:
ng serve
como usuario ldac se intento modificar componentes de una sala, nombre, foto y descripcion de las mismas con resultado exitoso.
se probo el caso en tratar de desactivar una sala que tiene reservas activas y se muestra el error correspondiente.
se intento modificar componentes con numeros muy grandes para observar comportamiento, se muestra un error y se reinicia la vista.

manuelitod committed:
Administracion de una sala.

Como administrador ahora se puede editar la imagen, los datos y los componentes de una sala.

manuelitod committed:
Administracion de una sala.

Como administrador ahora se puede editar la imagen, los datos y los componentes de una sala.

manuelitod committed:
Merge branch 'feature/admin-room' of github.com-manuelitod:PS1115-Skala/frontend-v2 into feature/admin-room

manuelitod committed:
Actualizado metodo para agregar imagen y mensaje de exito al guardar una sala sin componentes

dsegura95 commented on src/app/features/dashboard/components/dialogs/dialog-upload-image.component.ts:
```diff
@@ -0,0 +1,101 @@
+
+@Component({
+  selector: "app-dialog-upload-image",
+  template: `
```
Diria mover esto a un html para mantener el estilo de como se han hecho las cosas.

manuelitod replied:
Done

dsegura95 commented on src/app/features/dashboard/components/dialogs/dialog-upload-image.component.ts:
```diff
@@ -0,0 +1,101 @@
+      </button>
+    </mat-dialog-actions>
+  `,
+  styles: [
```
Lo mismo esto ponerlo en un .css

dsegura95 commented on src/app/features/dashboard/components/dialogs/dialog-upload-image.component.ts:
```diff
@@ -0,0 +1,101 @@
+    this.croppedImage = event.base64;
+  }
+  imageLoaded() {
+    // show cropper
```
Estas funciones vacias ??

dsegura95 commented on src/app/features/dashboard/pages/room-admin/room-admin.component.html:
```diff
@@ -0,0 +1,249 @@
+                  class="btn btn-success pull-right"
+                  type="submit"
+                >
+                  Editar Sala
```
Cambiar por Guardar

dsegura95 reviewed (changes requested):
Hacer pull con develop antes de volver a hacer push y resolver conflicto

manuelitod committed:
Merge branch 'develop' of github.com-manuelitod:PS1115-Skala/frontend-v2 into feature/admin-room

manuelitod committed:
Comentarios de PR.

Arreglado conflicto con package.json de develop.
Se crearon archivos html y css del dialog de upload image.
Se eliminaron funciones innecesarias del upload image.
Se agrego un fix para recargar imagen de cambio de sala al actualizarla.
